src/Fetch_Abort.res

```diff
@@ -1,10 +1,39 @@
+module AbortEvent = {
+  type t
+}
+
 module Signal = {
   type t
 
+  /**
+  https://dom.spec.whatwg.org/#dom-abortsignal-any
+  this is new as of 2024
+   */
+  @scope("AbortSignal")
+  external any: array<t> => t = "any"
+
+  @scope("AbortSignal")
+  external timeout: int => t = "timeout"
+
+  @scope("AbortSignal")
+  external abort: 'reason => t = "abort"
+
   @get external aborted: t => bool = "aborted"
   @get @return(nullable) external reason: t => option<_> = "aborted"
```

Comment:
Could you also rename this type variable to `'reason` to be consistent with the rest?
```suggestion
  @get @return(nullable) external reason: t => option<'reason> = "aborted"
```